model/refArchitecture.js

```diff
@@ -0,0 +1,29 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var refArchSchema = new Schema({
+    name :{
+        type: String,
+        unique : false,
+        required : true
+    },
+    refArch_id :{
+        type: String,
+        unique : true,
+        required : true
+    },
```

Comment:
let's add one more field called `type`. there can be reference architectures for application, infrastructure, data pipelines or devsecops.